### DISCUSSION TITLE
"RGB_colourspace_pointer_gamut_coverage_MonteCarlo" is not defined?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1110] [Q&A] [answered]
```
import numpy as np
from colour.models import RGB_COLOURSPACE_sRGB as sRGB

prng = np.random.RandomState(2)
RGB_colourspace_pointer_gamut_coverage_MonteCarlo(
    sRGB, 10e3, random_state=prng
)

```

But when I _python3 percent_of_pointers.py_

I get 

**Traceback (most recent call last):
  File "percent_of_pointers.py", line 5, in <module>
    RGB_colourspace_pointer_gamut_coverage_MonteCarlo(
NameError: name 'RGB_colourspace_pointer_gamut_coverage_MonteCarlo' is not defined**

Have I done something wrong?  The imports work fine inside Python.  I'm running colour 0.4.2, numpy 1.24.2, and Python 3.10.10.

- J

**Comment by thomasmansencal** [comment, marked as answer]
Hi @blue-j,

Can you verify that you have imported the object, reading your script it is not!

Cheers,

Thomas

**Reply by blue-j** [reply]
Oh dear, I thought what is shown was correct... Could I trouble you to give some guidance?  Much thanks to you, Thomas!

-J

**Reply by thomasmansencal** [reply]
```python
from colour import RGB_colourspace_pointer_gamut_coverage_MonteCarlo
```
